compose.yaml

```diff
@@ -4,7 +4,6 @@ services:
     build:
       context: .
       dockerfile: Dockerfile
-    runtime: nvidia
```

Comment:
We should not use the `--runtime=nvidia` option because it is a deprecated method since NVIDIA Container Toolkit 2.2.1. Instead, we use `--gpus`.

see https://qiita.com/ksasaki/items/b20a785e1a0f610efa08 (in Japanese)